tests/test_buddycheck.py

```diff
@@ -0,0 +1,162 @@
+import unittest
+from unittest.mock import patch
+import pandas as pd
+import tempfile
+import os
+from cautiousrobot import BuddyCheck
+from cautiousrobot.utils import process_csv
+
+class TestBuddyCheck(unittest.TestCase):
+    def setUp(self):
+        self.buddy_check = BuddyCheck()
+        self.buddy_check_filename = BuddyCheck(buddy_id='filename')
```

Comment:
```suggestion
        self.buddy_check_filename = BuddyCheck(buddy_id='filename')
        self.buddy_check_id_col = BuddyCheck(buddy_id = "filename", buddy_col = "sha256")
```
Add check for the other potential initialization parameters (overwriting defaults)?